model/src/cfd.rs

```diff
@@ -60,14 +61,12 @@ use std::ops::RangeInclusive;
 use std::str;
 use time::Duration;
 use time::OffsetDateTime;
-use uuid::adapter::Hyphenated;
 use uuid::Uuid;
 
 pub const CET_TIMELOCK: u32 = 12;
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, sqlx::Type)]
-#[sqlx(transparent)]
```

Comment:
won't this make it harder to debug things manually in the DB, if we only add hyphens in `Display` trait?